ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/aaa.md

```diff
@@ -125,10 +125,11 @@ radius-server host 10.10.10.158 vrf default key 7 071B245F5A
 ### AAA Server Groups Device Configuration
 
 ```eos
-!
 aaa group server tacacs+ TACACS
    server 10.10.10.157 vrf mgt
+!
```

Comment:
When pasting this config on a device I get the `!` before the first line, and no separator between the servers.

Reply:
Interesting.  I am running 4.25.4M, and I get the exclamation point in between. I just tried with 4.27.3F, here is the output:
```eos
!
aaa group server tacacs+ TACACS1
   server 10.10.10.157 vrf mgt
!
aaa group server tacacs+ TACACS2
   server 10.10.10.152 vrf mgt
!
aaa group server tacacs+ TACACS3
   server 10.25.200.69
!
aaa group server tacacs+ TACACS4
   server 192.168.100.69
!
```